src/libs/actions/IOU.ts

```diff
@@ -823,6 +823,22 @@ function buildOnyxDataForTrackExpense(
     const isScanRequest = TransactionUtils.isScanRequest(transaction);
     const clearedPendingFields = Object.fromEntries(Object.keys(transaction.pendingFields ?? {}).map((key) => [key, null]));
     const optimisticData: OnyxUpdate[] = [];
+    const isDistanceRequest = TransactionUtils.isDistanceRequest(transaction);
```

Comment:
👋 Could you please arrange `isDistanceRequest` right under `isScanRequest` so we have them grouped nicely ?

Reply:
i updated